cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -737,55 +737,106 @@
   <!-- Kafka Server Configuration -->
 
   <property>
-    <name>kafka.bind.address</name>
-    <value>0.0.0.0</value>
+    <name>kafka.log.retention.hours</name>
+    <value>${kafka.server.log.retention.hours}</value>
     <description>
-      CDAP Kafka service bind address
+      The number of hours to keep a log file before deleting it (deprecated: replaced with
+      kafka.server.log.retention.hours)
     </description>
   </property>
 
   <property>
-    <name>kafka.bind.port</name>
-    <value>9092</value>
+    <name>kafka.num.partitions</name>
+    <value>${kafka.server.num.partitions}</value>
     <description>
-      CDAP Kafka service bind port
+      Default number of partitions for a topic
+    </description>
+  </property>
+
+  <property>
+    <name>kafka.seed.brokers</name>
+    <value>127.0.0.1:9092</value>
```

Comment:
Right; this value isn't fed to Kafka server; It's used by cdap to talk to the kafka server.